tools/vars/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+collection_path: /your/local/collection/path/
+collections:
+  - amazon/aws
+  - cloud/aws_ops
+ansible_lint_version: "v6.21.0"
```

Comment:
We need to update this to at least 6.22.1. There is a bug in earlier versions that make it always use `main`.